CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
 ### Fixes
 
+* `[mock]` Add backticks support (\`\`) to `mock` a certain package via the `__mocks__` folder.([#5426](https://github.com/facebook/jest/pull/5426))
```

Comment:
Should this be `[jest-mock]`?

Reply:
You're right. I meant to fix that, forgot, though. Thanks!